src/pages/new/index.js

```diff
@@ -111,9 +111,9 @@ export default () => (
       quotes={[
         {
           quote:
-            "We've heard from San Franciscans, including as part of our recent strategic planning community outreach that it's hard to know where to go when you're looking for supportive services. It's hard to know where to start, and hard to navigate different resources to find accurate information. We're glad to be supporting ShelterTech and the broader community to build SF Service Guide - a one-stop, reliable place to help connect residents to the services they need.",
+            "We’ve heard from San Franciscans, including as part of our recent strategic planning community outreach that It’s hard to know where to go when you’re looking for supportive services. It’s hard to know where to start, and hard to navigate different resources to find accurate information. We’re glad to be supporting ShelterTech and the broader community to build SF Service Guide – a one-stop, reliable place to help connect residents to the services they need.",
```

Comment:
You ~accidentally~ capitalized one of the "It's" in the quote. It's the first one, in the middle of:

> strategic planning community outreach that <b>It’s</b> hard to know

Double checking the Figma design, I see that it also has the capitalization error. Can you leave the the "it's" uncapitalized here, since I'm pretty sure it's a typo in the design?

Reply:
Yes! Thanks for noticing this!!